### PR TITLE
Instructor: recycle bin feature: add instructions for managing deleted sessions in instructor help page #8964

### LINKED ISSUE #8964: Instructor: recycle bin feature: add instructions for managing deleted sessions in instructor help page
**Environment:**
`recycling-bin` branch

### PR DESCRIPTION
**Pull request by dalessr**
Fixes #8964 

**Current UI:**
8965-image-1

8965-image-2

8965-image-3

### CODE REVIEW

**Comment by dalessr** [comment]
@joanneong @darrenwee Ready for review. Can only look at changes in `instructorHelpSessions.jsp` if PR #8907 has yet to be merged.

**Comment by dalessr** [comment]
@darrenwee A gentle reminder to review this, thanks!

File: src/main/java/teammates/ui/pagedata/InstructorFeedbackSessionsPageData.java [hunk 1]
```diff
@@ -324,4 +347,77 @@ public void setUsingAjax(boolean isUsingAjax) {
...
+            ElementTag restoreButton = createButton("Restore", "btn btn-default btn-xs t_session_restore" + idx, "",
+                    restoreLink, Const.Tooltips.FEEDBACK_SESSION_RESTORE, !hasRestorePermission);
+
+            String deleteLink = getInstructorFeedbackDeleteRecoverySessionLink(session.getCourseId(),
+                    session.getSessionName());
+            Boolean hasDeletePermission = instructorsForCourses.get(session.getCourseId()).isAllowedForPrivilege(
+                    Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
+            ElementTag deleteButton = createButton("Delete Permanently", "btn btn-default btn-xs "
+                            + "t_session_delete" + idx, "fsDeleteLink", deleteLink, Const.Tooltips.FEEDBACK_SESSION_DELETE,
+                    !hasDeletePermission);
```
Comment (damithc) [inline src/main/java/teammates/ui/pagedata/InstructorFeedbackSessionsPageData.java]: This indentation doesn't look right. extract 2nd param as a local variable?

File: src/main/java/teammates/ui/pagedata/InstructorFeedbackSessionsPageData.java [hunk 2]
```diff
@@ -324,4 +347,77 @@ public void setUsingAjax(boolean isUsingAjax) {
...
+                    this.getInstructorCourseStatsLink(session.getCourseId()),
+                    actionsParam);
+            recoveryFeedbackSessionsTable.getRows().add(row);
+        }
+
+        return recoveryFeedbackSessionsTable;
+    }
+
+    private ElementTag createButton(String content, String buttonClass, String id, String href, String title,
+                                    boolean isDisabled) {
```
Comment (damithc) [inline src/main/java/teammates/ui/pagedata/InstructorFeedbackSessionsPageData.java]: Shouldn't this be 8 space indentation?

File: src/main/java/teammates/ui/template/RecoveryFeedbackSessionsTableRow.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+    private String deletedTimeDateStamp;
+    private String deletedTimeFullDateTimeString;
+    private String href;
+    private List<ElementTag> actions;
+
+    public RecoveryFeedbackSessionsTableRow(String courseIdParam, String sessionNameParam,
+                                   String createdTimeDateStringParam, String createdTimeDateStampParam,
+                                   String createdTimeFullDateTimeStringParam, String deletedTimeDateStringParam,
+                                   String deletedTimeDateStampParam, String deletedTimeFullDateTimeStringParam,
+                                   String href, List<ElementTag> actionsParam) {
```
Comment (damithc) [inline src/main/java/teammates/ui/template/RecoveryFeedbackSessionsTableRow.java]: 8 space indentation?

**Comment by dalessr** [comment]
This PR is to be merged after #8907 has been merged.

**Pull request merged**